lib/rules/unit-no-unknown/index.js

```diff
@@ -129,7 +130,7 @@ function rule(actual, options) {
 		}
 
 		root.walkAtRules((atRule) => {
-			if (!/^media$/i.test(atRule.name) && !atRule.variable) {
+			if (!/^media$/i.test(atRule.name) && !isStandardSyntaxAtRule(atRule)) {
```

Comment:
[suggestion] Can we move the position of `/^media$/i` as below? I think it's more readable. 😃 

```js
		root.walkAtRules(/^media$/i, (atRule) => {
			if (!isStandardSyntaxAtRule(atRule)) {

```

Reply:
Yep, we should be using the `walk*` filter when available. I think most rules do, but it looks like this one must have slipped through at some point.

Reply:
@ybiquitous @jeddy3 thanks a lot guys for your review :)